### DISCUSSION TITLE
flux2 gives panic: runtime error: index out of range [0] with length 0 when bootstraping bitbucket server

### DISCUSSION THREAD

**Discussion by vijaymailb** [fluxcd/flux2#2332] [Q&A] [answered]
Hello All,

I am trying to bootstrap bitbucket server with existing repository. Please find my command below.

flux bootstrap bitbucket-server --owner=project_name --username=user_name --repository=repo_name --branch=branch_name --hostname=bitbucket_fqdn --token-auth --verbose
Please enter your Bitbucket personal access token (PAT):

After ran the above command, hit with this error.

► connecting to bitbucket_fqdn
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/fluxcd/go-git-providers/stash.(*ProjectsService).Get(0xc0002060c0, {0x2c07838, 0xc000144fc0}, {0x7ffeefbffab0, 0x3})
	/home/runner/go/pkg/mod/github.com/fluxcd/go-git-providers@v0.5.2/stash/projects.go:171 +0x496
github.com/fluxcd/go-git-providers/stash.(*OrganizationsClient).Get(0xc0002060f8, {0x2c07838, 0xc000144fc0}, {{0xc000d53028, 0x17}, {0x7ffeefbffab0, 0x3}, {0x0, 0x0}, {0x0, ...}})
	/home/runner/go/pkg/mod/github.com/fluxcd/go-git-providers@v0.5.2/stash/client_organizations.go:43 +0x10c
github.com/fluxcd/flux2/internal/bootstrap.(*GitProviderBootstrapper).getOrganization(0xc00043d040, {0x2c07838, 0xc000144fc0}, {0x0, 0x0, 0x0})
	/home/runner/work/flux2/flux2/internal/bootstrap/bootstrap_provider.go:400 +0x22e
github.com/fluxcd/flux2/internal/bootstrap.(*GitProviderBootstrapper).reconcileOrgRepository(0xc00043d040, {0x2c07838, 0xc000144fc0})
	/home/runner/work/flux2/flux2/internal/bootstrap/bootstrap_provider.go:276 +0x14d
github.com/fluxcd/flux2/internal/bootstrap.(*GitProviderBootstrapper).ReconcileRepository(0xc00043d040, {0x2c07838, 0xc000144fc0})
	/home/runner/work/flux2/flux2/internal/bootstrap/bootstrap_provider.go:222 +0x39
github.com/fluxcd/flux2/internal/bootstrap.Run({_, _}, {_, _}, {_, _}, {{0x2939140, 0x28}, {0xc000626a20, 0x7}, ...}, ...)
	/home/runner/work/flux2/flux2/internal/bootstrap/bootstrap.go:85 +0xaf
main.bootstrapBServerCmdRun(0x3b9df60, {0x28f880a, 0x7, 0x7})
	/home/runner/work/flux2/flux2/cmd/flux/bootstrap_bitbucket_server.go:278 +0x1565
github.com/spf13/cobra.(*Command).execute(0x3b9df60, {0xc00035d3b0, 0x7, 0x7})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x3b95860)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
main.main()
	/home/runner/work/flux2/flux2/cmd/flux/main.go:145 +0x32

**Comment by souleb** [comment, marked as answer]
The project `--owner=project_name ` does not exist. You should enter an existing project. It should not panic though. We will fix this.
